tests/backup/backup_helper.go

```diff
@@ -6302,3 +6302,26 @@ func isStorageClassPresent(storageClassName string) (bool, error) {
 	}
 	return false, nil
 }
+
+//dumpMongodbCollectionOnConsole Dump the collection mentioned on the console for debugging
```

Comment:
nit: there needs to be a space after `//` else GoLand will format it
nit: `Dumps`

Reply:
Done.